spec/factories/dttp/providers.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :dttp_provider, class: "Dttp::Provider" do
+    sequence :name do |n|
+      "Provider #{n}"
+    end
+    dttp_id { SecureRandom.uuid }
+    ukprn { Faker::Alphanumeric.alphanumeric(number: 3).upcase }
```

Comment:
pretty sure that should be 8
```suggestion
    ukprn { Faker::Alphanumeric.alphanumeric(number: 8).upcase }
```

@misaka to confirm, please get involve.

Reply:
I picked it up from https://github.com/DFE-Digital/register-trainee-teachers/blob/afb651222bbd7df9a1819328873212cb13e2ba04/spec/factories/providers.rb#L9
Noticed that in development I'm getting alphanumeric values of lengths between 8-10

Reply:
@misaka @gpeng 
>  Noticed that in development I'm getting alphanumeric values of lengths between 8-10

What is it in production? What is the rules/integrity we need to retain/enforce?

cc @avinhurry

Reply:
In my not all that scientific check on production DTTP all UKPRNs look to be 7, 8 or 9 digit numbers.